crates/storage/src/vm_storage.rs

```diff
@@ -58,8 +58,19 @@ use fuel_vm_private::{
 };
 use itertools::Itertools;
 use primitive_types::U256;
+
+#[cfg(feature = "std")]
 use std::borrow::Cow;
 
+#[cfg(not(feature = "std"))]
+use alloc::borrow::Cow;
```

Comment:
```suggestion
use alloc::borrow::Cow;
```

nit: I'm pretty sure you can just do this instead

Reply:
It will work if `alloc` feature is enabled